src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.html

```diff
@@ -238,205 +262,86 @@ <h4 *ngIf="this.rangeFilter" class="d-flex align-items-center">
                     </ng-template>
                 </ngx-datatable-column>
 
-                <ngx-datatable-column prop="durationInMinutes" [resizeable]="false" [sortable]="true" [minWidth]="100" [width]="100">
+                <ngx-datatable-column prop="" [sortable]="true" [minWidth]="80" [width]="90">
```

Comment:
Specifying `width` probably immediately overwrites the `minWidth` property.

Reply:
This is done in many components. Afaik the width only specifies the preferred width, but in case of small screens, the width can be shrunken down to the min width